WordPress/Classes/Models/ReaderSearchTopic.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+@objc public class ReaderSearchTopic : ReaderAbstractTopic
+{
+    override public class var TopicType: String {
```

Comment:
Sure! (since we don't have a standard here) we can either capitalize them all, or not, in another PR!